### PR TITLE
process outgoing telegrams in devices

### PR DESCRIPTION
**Pull request by farmio**
process outgoing telegrams in all devices - regarding #393 
This enables xknx-devices that share a specific GA (without state_address) to be updated when a new value is sent form xknx.

I'm not sure if this would break anything - I guess it does. On my installation everything seems to work, but I don't use fancy binary_sensors with count and actions etc...

Updating the tests for this change will be non-trivial 😬

### CODE REVIEW

File: xknx/core/telegram_queue.py [hunk 1]
```diff
@@ -140,7 +145,7 @@ async def process_telegram_incoming(self, telegram):
                 ret = await telegram_received_cb.callback(telegram)
                 if ret:
                     processed = True
-
+        # TODO: why don't we process every incoming telegram by device?
```
Comment (farmio) [inline xknx/core/telegram_queue.py:148]: The callbacks usually come from value_reader. I think if a knx-device responds, we should update all fo our devices that use the GA. 
Another source for these callbacks would be HAs fire_event where we use 
```python
        # False signals XKNX to proceed with processing telegrams.
        return False
```
so `processed` is False.

**Review by marvin-w — COMMENTED** [review]
Tested with binary sensor count == 1 and count == 2, with ignore_internal_state on and off. Seems to work fine. I used the same set of tests that I used for the refactoring.

**Review by marvin-w — APPROVED** [review]
Tested and works fine.

**Comment by farmio** [comment]
this should
fix https://github.com/home-assistant/core/issues/40351
fix https://github.com/home-assistant/core/issues/40316
fix https://github.com/home-assistant/core/issues/40359
closes #393

**Pull request merged**